literature.bib

```diff
@@ -4480,6 +4600,77 @@ @article{TKB+:SCP14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2013 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{SLSA:SIGPLAN13,
+	author = {Schulze, Sandro and Liebig, J\"{o}rg and Siegmund, Janet and Apel, Sven},
+	title = {{Does the Discipline of Preprocessor Annotations Matter?: A Controlled Experiment}},
+	journal = SIGPLAN,
+	volume = {49},
+	number = {3},
+	year = 2013,
+	issn = {0362-1340},
+	pages = {65--74},
+	numpages = {10},
+	url = {http://doi.acm.org/10.1145/2637365.2517215},
+	doi = {10.1145/2637365.2517215},
+	acmid = {2517215},
+	publisher = ACM,
+	address = NY,
+	keywords = {c preprocessor, controlled experiment, disciplined annotations, program comprehension, variability},
+}
+
+@inproceedings{KKT:ACM13,
+	author = {Timo {Kehrer} and Udo {Kelter} and Gabriele {Taentzer}},
+	booktitle = ASE,
+	publisher = ACM,
+	title = {{Consistency-Preserving Edit Scripts in Model Versioning}},
```

Comment:
address of ACM missing, but is always the same

Reply:
added NY